.kitchen.docker.yml

```diff
@@ -19,3 +19,13 @@ platforms:
     run_list: yum-epel::default
   - name: centos-7
     run_list: yum-epel::default
+
+suites:
+  - name: default
+    run_list: omnibus::default
+
+  #
+  # Build
+  #
+  - name: omnibus_build_execute
+    run_list: omnibus_build::execute
```

Comment:
Why are we recreating the suites in here? This `.kitchen.docker.yml` file is meant to laid over the top of the existing `.kitchen.yml` file. It can be easily "activated" with `KITCHEN_LOCAL_YAML`:
https://github.com/chef-cookbooks/omnibus/blob/master/.travis.yml#L56